website/static/js/contribManager.js

```diff
@@ -182,50 +222,79 @@ var MessageModel = function(text, level) {
 
 };
 
-var ContributorsViewModel = function(contributors, adminContributors, user, isRegistration) {
+var ContributorsViewModel = function(contributors, adminContributors, user, isRegistration, table, adminTable) {
 
     var self = this;
 
     self.original = ko.observableArray(contributors);
 
+    self.table = $(table);
+    self.adminTable = $(adminTable);
+
+    self.permissionDict = {
```

Comment:
Could be renamed (there are no "dicts" in JS). Maybe `permissionMap`. This isn't a big deal though.